client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -34,9 +34,8 @@ export function getEnhancedTasks(
 				case 'setup_newsletter':
 					taskData = {
 						title: translate( 'Personalize Newsletter' ),
-						keepActive: true,
```

Comment:
I think this probably got lost in rebasing. I updated keepActive to be true in [this PR](https://github.com/Automattic/wp-calypso/pull/68685). Personalize Newsletter is now an active link, which enabled users to edit their site title/tageline/icon on the post-setup page.

Since keepActive is no longer valid, we'd just want to change `disabled` to be false for this task in tasks.tsx. That will reenable the link here.

Reply:
This is resolved.